python-summary-statistics/statistics.py

```diff
@@ -1,71 +1,159 @@
 #!/usr/bin/env python3
 
-from io_helper import io_helper
+from mip_helper import io_helper, shapes
 
+import argparse
+import itertools
 import logging
-import pandas
-
-
-OUTPUT_SCHEMA = {
-        'schema': {
-            'field': [
-                {'name': 'index', 'type': 'string'},
-                {'name': 'count', 'type': 'object'},
-                {'name': 'mean', 'type': 'number'},
-                {'name': 'std', 'type': 'number'},
-                {'name': 'min', 'type': 'number'},
-                {'name': '25%', 'type': 'number'},
-                {'name': '50%', 'type': 'number'},
-                {'name': '75%', 'type': 'number'},
-                {'name': 'max', 'type': 'number'}
-            ]
-        },
-        'data': []
-    }
-
-
-def main():
-    # Configure logging
-    logging.basicConfig(level=logging.INFO)
-
+import json
+import copy
+import numpy as np
+import pandas as pd
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+
+# TODO: use something more fancy like schematics instead of dict?
+OUTPUT_SCHEMA_INTERMEDIATE = {
```

Comment:
Should I stick to the original schema? I've replaced `count` by `freq` and changes its type, so I'm a bit worried it might break things in production.